api/external/cfgmgmt/cfgmgmt.proto

```diff
@@ -32,7 +32,7 @@ import "components/automate-grpc/protoc-gen-policy/iam/annotations.proto";
 
 service ConfigMgmt {
   /*
-  GetNodes
+  List Checked-in Nodes
```

Comment:
get is a little bit different than list, idk if we care

Reply:
The use of "list" is based off of the authorization action being "list" per the lines below.